### ISSUE TITLE
Video portrait segmentation is not unstable，often flickering when use unet model

### ISSUE THREAD

**Issue by MHX1203** [AntiAegis/Semantic-Segmentation-PyTorch#44]
Video portrait segmentation is not unstable， often flickering when use unet model，and loss used crossentropy. i am wonder I what causes the flicker and whether it is related to training loss。

I am looking forward to answer。
